.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: 
+  - rubocop-rails
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'bin/**/*'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'spec/spec_helper.rb'
+    - 'node_modules/**/*'
+    - 'tmp/*'
+    - 'config/initializers/**/*'
+    - 'lib/tasks/*'
+    - 'Guardfile'
```

Comment:
I'm curious what your criteria for exclusions were?

Some of these, i would love to have `rubocop` weigh in on:
* `config/initializers` because i'm not sure we should be doing anything complicated in there
* `lib/tasks` because some rake tasks can get pretty unwieldy if they aren't kept in check!
* `spec_helper` is another one that can get bloated

Others i don't feel a particular need to have coverage for, but am curious what about them was a problem?
* `db/`
* `bin/`
* `Gemfile`

Lastly, there are a few i _think_ we can drop from this list:
* `node_modules` is excluded by [default](https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml)
* Same for `tmp`
* I don't think our project has a `Rakefile`
* We _do_ have a `Guardfile`, but we shouldn't -- that was an oversight on my part when i removed the guard gem 🤭 

Sorry if that is overly picky! None of it is a major concern, for sure.

Reply:
I just pulled some standard .rubocop.yml files and used the ones that I have seen the most consistently. I have no issues with changing anything. No worries!

Reply:
We do have a Rakefile though, but like I said I am fine changing anything.

Reply:
node_modules are not excluded by default, I put that one in there myself.